scripts/skip-upgrade-tests.txt

```diff
@@ -165,4 +165,6 @@ vcd.TestAccVcdVAppVmDhcpWait.tf v3.0.0 "Terraform 0.13+ incompatibility"
 vcd.TestVappVmDS.tf v3.0.0 "Terraform 0.13+ incompatibility"
 vcd.TestAccVcdVappNetworkDS.tf v3.0.0 "Terraform 0.13+ incompatibility"
 vcd.TestAccVcdVappOrgNetworkDS.tf v3.0.0 "Terraform 0.13+ incompatibility"
-vcd.TestAccVcdEdgeGatewaySettingsFull.tf v3.0.0 "Terraform 0.13+ shows expected diff for 'output', but it cannot be ignored"
\ No newline at end of file
+vcd.TestAccVcdEdgeGatewaySettingsFull.tf v3.0.0 "Terraform 0.13+ shows expected diff for 'output', but it cannot be ignored"
+vcd.ResourceSchema-vcd_vapp_vm.tf v3.1.0 "Schema changes in VM due to standalone VM implementation"
+vcd.ResourceList-resources-resources.tf v3.1.0 "v3.2.0 introduced new options and this test complains on output diff"
```

Comment:
Makes sense because schema was altered.

```
Changes to Outputs:
  ~ resources = {
      ~ attributes       = [
          + {
              + computed    = false
              + description = "The vApp this VM belongs to - Required, unless it is a standalone VM"
              + name        = "vapp_name"
              + optional    = false
              + required    = true
              + sensitive   = false                                                                                                                                  + type        = "string"
            },
          - {
              - computed    = false
              - description = "The vApp this VM belongs to"
              - name        = "vapp_name"
              - optional    = false
              - required    = true
              - sensitive   = false
              - type        = "string"
            },
          + {
              + computed    = true
              + description = "Type of VM: either 'vcd_vapp_vm' or 'vcd_vm'"
              + name        = "vm_type"
              + optional    = false
              + required    = false
              + sensitive   = false
              + type        = "string"
            },
            # (28 unchanged elements hidden)
        ]
        # (4 unchanged elements hidden)
    }
```